### DISCUSSION TITLE
Why does it need all these permissions?

### DISCUSSION THREAD

**Discussion by kidroca** [dsoftwareinc/ghactions-manager#87] [Q&A] [answered]
Can you explain why certain permissions are required to activate this package?
- repo (r/w access to all my repositories)
- gist (r/w access to my gists)
- read:org (read access to organisations and team members)
- workflow (r/w access to workflows)

![Screenshot 2023-04-10 at 19 10 34](https://user-images.githubusercontent.com/12156624/230943024-5c6e9e35-e510-42c7-9cc3-d851e769dfdb.png)

**Comment by cunla** [comment, marked as answer]
Hi,

This is not for this plugin directly, it is for intellij github.
You can generate a token externally with workflows access only and paste it in the plugin settings.
